pkg/cache/v3/snapshot.go

```diff
@@ -137,9 +137,9 @@ func NewSnapshotWithResources(version string, resources SnapshotResources) Snaps
 	return out
 }
 
-type ResourceWithTtl struct {
+type ResourceWithTtl struct { // nolint:golint,revive
```

Comment:
Same here

Reply:
Removed this, since it is unused in this repository.